Reports/report-last30DaysClosedIncidentsReport.json

```diff
@@ -2,405 +2,628 @@
+            "fromValue": 30,
+            "by": "day"
+          },
+          "fromDate": "2017-12-24T22:00:00Z",
+          "toDate": null
+        }
+      },
+      "type": "chart",
+      "title": "Analyst"
     },
     {
-      "type": "header",
-      "data": "Closed incidents in the last 30 days",
       "layout": {
-        "rowPos": 9,
-        "columnPos": 1,
-        "style": {
-          "textAlign": "left",
-          "fontSize": 18
+        "tableColumns": [
+          "name",
+          "occurred",
+          "type",
+          "owner",
+          "severity",
+          "status",
+          "closed",
+          "closeNotes"
+        ],
+        "classes": "striped stackable small very compact",
+        "i": "456e8280-0064-11e8-831f-e7429aff1c69",
+        "rowPos": 6,
+        "columnPos": 0,
+        "w": 12,
+        "h": 2
+      },
+      "query": {
+        "type": "incident",
+        "filter": {
+          "query": "status:Closed",
+          "period": {
+            "byFrom": "days",
+            "fromValue": 30,
+            "by": "day"
+          },
+          "fromDate": "2017-12-24T22:00:00Z",
+          "toDate": null
         }
-      }
+      },
+      "type": "table",
+      "title": "Closed incidents in the last 30 days"
     },
     {
-      "type": "table",
+      "layout": {
+        "i": "5c9c5030-00ec-11e8-9b6f-436025e19450",
+        "rowPos": 1,
+        "columnPos": 0,
+        "w": 12,
+        "h": 1
+      },
       "query": {
         "type": "incident",
         "filter": {
-          "status": [2, 3],
-          "andOp": true,
+          "query": "status:Closed",
           "period": {
-            "by": "day",
+            "byFrom": "days",
             "fromValue": 30,
-            "toValue": 0,
-            "field": "closed"
-          }
+            "by": "day"
+          },
+          "fromDate": "2017-12-24T22:00:00Z",
+          "toDate": null
         }
       },
-      "layout": {
-        "rowPos": 10,
-        "columnPos": 1,
-        "tableColumns": ["Name", "Occurred", "Type", "Owner", "Severity", "Status", "Closed", "Close notes"],
-        "readableHeaders": {
-          "name": "Name",
-          "occurred": "Occurred",
-          "type": "Type",
-          "owner": "Owner",
-          "severity": "Severity",
-          "status": "Status",
-          "closed": "Closed",
-          "closeNotes": "Close notes"
-        },
-        "classes": "striped stackable small very compact"
-      }
+      "data": "# Last 30 days closed incidents: {0}",
+      "type": "placeholder",
+      "title": "last24HoursClosedIncidentsReport Headline"
     }
-  ]
-}
+  ],
+  "recipients": [],
+  "orientation": "",
+  "paperSize": "",
+  "runOnce": false,
+  "latestReportName": "",
+  "latestReportTime": "0001-01-01T00:00:00Z",
+  "latestScheduledReportTime": "0001-01-01T00:00:00Z",
+  "nextScheduledTime": "0001-01-01T00:00:00Z",
+  "latestReportUsername": "",
+  "decoder": null,
```

Comment:
please make sure to not remove the decoder in all reports..

Reply:
This is how I got it from the "Preview" command

Reply:
it is possible you have done it before we added it. you can create a new report and check it. Just remove the changes made for decoder value